pydocx/openxml/wordprocessing/textbox.py

```diff
@@ -0,0 +1,23 @@
+# coding: utf-8
+from __future__ import (
+    absolute_import,
+    print_function,
+    unicode_literals,
+)
+
+from pydocx.models import XmlModel, XmlCollection
+
+
+class TxBxContent(XmlModel):
+    XML_TAG = 'txbxContent'
+    children = XmlCollection(
+        'wordprocessing.Paragraph',
+    )
+
+
+class Textbox(XmlModel):
```

Comment:
This guy should really live in `vml` instead of `wordprocessing` :( I'll handle this after the code review.